templates/CRM/Block/RecentlyViewed.tpl

```diff
@@ -25,30 +25,34 @@
 *}
 {* Displays recently viewed objects (contacts and other objects like groups, notes, etc. *}
 <div id="crm-recently-viewed" class="left crm-container">
-    <ul>
+  <ul>
     {foreach from=$recentlyViewed item=item}
-         <li class="crm-recently-viewed" ><a  href="{$item.url}" title="{$item.title|escape:'html'}">
-         {if $item.image_url}
-            <span class="icon crm-icon {if $item.subtype}{$item.subtype}{else}{$item.type}{/if}-icon" style="background: url('{$item.image_url}')"></span>
-         {else}
-            <span class="icon crm-icon {$item.type}{if $item.subtype}-subtype{/if}-icon"></span>
-         {/if}
-         {if $item.isDeleted}<del>{/if}{$item.title|mb_truncate:25:"..":true}{if $item.isDeleted}</del>{/if}</a>
-         <div class="crm-recentview-wrapper">
-           <a href='{$item.url}' class="crm-actions-view">{ts}View{/ts}</a>
-         {if $item.edit_url}<a href='{$item.edit_url}' class="crm-actions-edit">{ts}Edit{/ts}</a>{/if}
-       {if $item.delete_url}<a href='{$item.delete_url}' class="crm-actions-delete">{ts}Delete{/ts}</a>{/if}
-         </div>
-       </li>
+      <li class="crm-recently-viewed">
+        <div class="crm-recentview-item">
```

Comment:
I wonder would it be nicer to wrap a DIV into A? In that case the area of click will cover the whole div, not just a caption (not a change request but a thought)